src/Components/Footer/Footer.css

```diff
@@ -1,6 +1,22 @@
 .siteInfo {
   width: 98.9vw;
+  background-color: black;
+  color: aliceblue;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   height: 8vh;
+  padding-top: 4px;
+  border-top: 1px solid blue;
```

Comment:
Put all these colors in your root variables

Reply:
Disabled the button until user enter his/her name. Thanks.

Reply:
Added CSS variables.